actors/miner/src/policy.rs

```diff
@@ -22,6 +23,11 @@ use super::{PowerPair, BASE_REWARD_FOR_DISPUTED_WINDOW_POST};
 /// Precision used for making QA power calculations
 pub const SECTOR_QUALITY_PRECISION: i64 = 20;
 
+/// The minimum sector lifetime to which the SDM applies.
+pub const SECTOR_DURATION_MULTIPLIER_START: ChainEpoch = 540 * EPOCHS_IN_DAY;
+/// The sector lifetime at which the SDM maxes out.
+pub const SECTOR_DURATION_MULTIPLIER_END: ChainEpoch = 5 * EPOCHS_IN_YEAR;
```

Comment:
Put these in the policy? All of our "power for sector" functions would need to take the policy, in that case.

Reply:
In policy is probably more standard at this point as it's the way we now manage most protocol constants.

Reply:
Yeah, it's just a really painful change. IMO, we should land this as-is then tackle that in a followup patch to see if it's worth it.